corpus/import_vars.clj

```diff
@@ -3,7 +3,11 @@
 (defn foo [_x]) ;; 1 arg
 
 (ns app.api
-  (:require [potemkin :refer [import-vars]]))
+  (:require
+   [clojure.walk]
+   [clojure.data]
+   [app.core]
```

Comment:
Required add the namespace now (like it's required to compile successfully this file on a REPL)